pkg/envfuncs/kind_funcs.go

```diff
@@ -53,6 +53,31 @@ func CreateKindCluster(clusterName string) env.Func {
 	}
 }
 
+// CreateKindClusterWithParam returns an env.Func that is used to
+// create a kind cluster that is then injected in the context
+// using the name as a key.
+//
+// NOTE: the returned function will update its env config with the
+// kubeconfig file for the config client.
+//
+func CreateKindClusterWithParam(clusterName string, image string, configFilePath string) env.Func {
```

Comment:
Since this is taking a kind config file, would a more descriptive name like `CreateKindClusterWithConfig` be a better name ?